src/server/game/Entities/Player/Player.h

```diff
@@ -1656,6 +1656,9 @@ class Player : public Unit, public GridObject<Player>
     void UpdateNextMailTimeAndUnreads();
     void AddNewMailDeliverTime(time_t deliver_time);
 
+    void SendUnlearnSpells();
+    static bool IsUnlearnSpellsPacketNeededForSpell(uint32 spellId);
```

Comment:
```suggestion
    static bool IsUnlearnNeededForSpell(uint32 spellId);
```
Not a fan of the name being as long as it is, feels unnecessary.